data_management/models.py

```diff
@@ -580,11 +580,11 @@ class ExternalObject(BaseModel):
       modelling pipeline.***
 
     ### Writable Fields:
-    `identifier`: Full URL of identifier (e.g. DataCite DOI) of the `ExternalObject`, unique in the context of the triple (`identifier`, `title`, `version`).  This needs to be specified only if `other_unique_name` is not.
+    `identifier`: Full URL of identifier (e.g. DataCite DOI) of the `ExternalObject`, unique in the context of the triple (`identifier`, `title`, `version`). At least one of `identifier` and `alternate_identifier` must be defined.
 
-    `other_unique_name`: Name of the `ExternalObject`, unique in the context of the triple (`other_unique_name`, `title`, `version`). This needs to be specified only if `identifier` is not.
+    `alternate_identifier`: Name of the `ExternalObject`, unique in the context of the triple (`alternate_identifier`, `title`, `version`).
```

Comment:
```suggestion
    `alternate_identifier`: Name of the `ExternalObject`, unique in the context of the quadruple (`alternate_identifier`, `alternate_identifier_type`, `title`, `version`). Unlike `identifier`, this is free text, not a url. For instance, it can be a locally unique name for a data resource within the domain of issue. It is associated with a `alternate_identifier_type` which describes its origin.
```